core/__init__.py

```diff
@@ -24,19 +24,14 @@
  See file LICENSE for more information.
 """
 
-from types import ModuleType
-from typing import Dict, Union, Callable
-from viur.core.config import conf
-from viur.core import request, utils
-from viur.core.i18n import initializeTranslations
-from viur.core import logging as viurLogging  # Initialize request logging
-from viur.core.session import GaeSession
-from viur.core.version import __version__
 import logging
 import webob
-
-# Copy our Version into the config so that our renders can access it
-conf["viur.version"] = tuple(__version__.split(".", 3))
+from types import ModuleType
+from typing import Callable, Dict, Union
+from viur.core import session, errors, i18n, request, utils
+from viur.core.config import conf
+from viur.core.tasks import TaskHandler, runStartupTasks
+from viur.core import logging as viurLogging  # Initialize request logging # noqa: E402
```

Comment:
It's now at the top of the file, so it should fit with E402 :)
```suggestion
from viur.core import logging as viurLogging  # Initialize request logging
```